dist/demo/demo.bundle.js

```diff
@@ -436,8 +436,15 @@ module.exports = reactProdInvariant;
 /***/ (function(module, exports, __webpack_require__) {
 
 "use strict";
+/*
```

Comment:
You might want to rebuild these dist files yourself as i might be using a different version of webpack causing all these extra changes..